src/utils/compositionUtils.js

```diff
@@ -10,7 +17,7 @@ export function extendArray(objects) {
   objects.unshift({});
 
   //Buid the combined object
-  let combinedObject = _.extend.apply(this, objects);
+  let combinedObject = assignIn.apply(this, objects);
```

Comment:
Nice! 
Per the docs. https://lodash.com/docs/#assignIn  extend is the alias so using the "real" is better.